ansible/inventory/group_vars/development/default.yml

```diff
@@ -0,0 +1,4 @@
+---
+docker_build: true
+quickfeed_endpoint: dev.itest.run
+ag_port: 8090
```

Comment:
Should be changed to `http_addr` or something, because it is really the `-http.addr` flag for the https endpoint of the echo framework. (The plan is to refactor away the echo framework at some point... but we will probably still need a `http.addr` flag.)